### ISSUE TITLE
outdated extension

### ISSUE THREAD

**Issue by afzelarshad13** [ClaudiuCreanga/magento2-store-locator-stockists-extension#54]
now with php8.1 and magento2.4.5, this extension is outdated.

**Comment by ClaudiuCreanga** [comment]
> now with php8.1 and magento2.4.5, this extension is outdated.

I'm out of magento and php world. the extension doesn't work anymore?

**Comment by ClaudiuCreanga** [comment]
I've transferred the repo to @motou who hopefully will have some time to invest in this extension and bring it up to date.

**Comment by afzelarshad13** [comment]
thanks man, appreciate it

**Comment by piontuso** [comment]
hi @motou ! Could you please update composer.json and add support for php 8.3 ? We want use this module on Magento 2.4.7-p2 php 8.3. 

Thanks in advance!
Maciej

**Comment by motou** [comment]
Yes sure, I hope that I can find some time this week to update the plugin dependencies.

**Comment by piontuso** [comment]
cool! very appreciate

**Comment by piontuso** [comment]
hi @motou  - did you have time to check my request ? I added PR to make it faster

**Comment by motou** [comment]
With the PR https://github.com/motou/magento2-store-locator-stockists-extension/pull/59, the extension supports PHP 8.4 already
